test/e2e/framework.go

```diff
@@ -119,7 +119,7 @@ const (
 	antreaControllerCovFile   = "antrea-controller.cov.out"
 	antreaAgentCovFile        = "antrea-agent.cov.out"
 	flowAggregatorCovFile     = "flow-aggregator.cov.out"
-	cpNodeCoverageDir         = "/tmp/antrea-e2e-coverage"
+	cpNodeCoverageDir         = "/tmp/coverage"
```

Comment:
I am not sure why do we need to change this directory name?
I feel the old name is more clear.

Reply:
Decided to use the same directory name throughout(pods, nodes, etc.). Will revert if it creates ambiguity.

Reply:
Yes it may create some ambiguity as the name is a bit too "generic", and this is the Node's filesystem